package.json

```diff
@@ -31,6 +31,8 @@
     "node": ">=18.0.0"
   },
   "dependencies": {
-    "gatsby-plugin-testing": "^0.3.5"
+    "gatsby-plugin-testing": "^0.3.5",
+    "lunr": "^2.3.9",
+    "react-modal": "^3.16.1"
```

Comment:
I think this needs to be in packages/gatsby-theme-project-portal/package.json rather than here

Reply:
done!